### PR TITLE
Release 2021 W50-52

### PR DESCRIPTION
**Pull request by extreme4all**

### CODE REVIEW

**Comment by Ferrariic** [comment]
Great job pulling this release together. I'll be able to review it in a bit.

**Comment by extreme4all** [comment]
PR should not be linked to a sprint, rather we link the issue to the sprint

File: api/Config.py [hunk 1]
```diff
@@ -20,6 +21,21 @@
...
 app = FastAPI()
 
+origins = [
+    "http://osrsbotdetector.com/",
+    "https://osrsbotdetector.com/",
+    "http://localhost",
+    "http://localhost:8080",
+]
+
+app.add_middleware(
```
Comment (Ferrariic) [inline api/Config.py:31]: Thanks for this, good idea.

File: api/database/database.py [hunk 2]
```diff
@@ -28,41 +27,34 @@ def __init__(self, engine_type: EngineType = EngineType.PLAYERDATA):
...
-            pool_pre_ping=True
-
+            pool_pre_ping=True,
+            pool_size=100, 
+            max_overflow=500,
+            pool_recycle=3600
         )
-  
-        self.session = sessionmaker(self.engine, class_=AsyncSession, expire_on_commit=False)
+        # self.engine.echo = True
```
Comment (Ferrariic) [inline api/database/database.py:35]: We can remove this comment, though not necessary. Not a critical issue.

**Review by Ferrariic — APPROVED** [review]

**Pull request merged**